### ISSUE TITLE
Allow users to deactivate restricted airspace

### ISSUE THREAD

**Issue by erikquinn** [openscope/openscope#1352]
Users should be able to turn restricted airspace on and off as they desire, with the exception of Prohibited Areas, which mist remain active at all times.

The best solution would be to allow them to turn it off globally, and if they want, to turn them on/off individually. But adding a simple on/off switch for everything would be a perfectly good resolution for the time being. And again, regardless of their choices, prohibited areas would remain hot (this will require some new logic to differentiate different _types_ of SUA).

**Comment by Fechulo** [comment]
It might be better to have this as an option in the airport file for each area. As far as I know, restricted areas are just as important as prohibited areas in most of Europe and deactivating them would be very unrealistic. If we add a key for each restricted airspace (something like `optional: true;`), then we can decide which areas can be deactivated and which cannot.

**Comment by guyndumais** [comment]
I agree with @Fechulo that turning off restricted areas would make the simulation unrealistic. This problem is easily reproduced on KDCA. Start airport, timewarp 60 and within 1-2 minutes an uncontrolled plane from WAVES CAPSS3 will cross p-56a (White House). It's a problem specific to KDCA, don't remember other airports having this problem.

**Comment by erikquinn** [comment]
Again, different types of restricted airspace go active/inactive all the time. It is this airspace only that would be able to be modified. For example, along that route is R6601, which is activated and deactivated regularly by notam. This one could be turned on/off. P56 is a prohibited area all day every day, and would not be editable.

The point is that we need to distinguish which ones can be activated/deactivated by the users, and which ones cannot, so we can achieve the maximum level of realism. Everything always being hot is actually less realistic, hence why we're taking the path of grouping different SUA.

The DCA arrivals being uncontrollable above your airspace and hitting P56 is a separate bug, and has no real relation to this feature request.
